src/System Application/App/File System/src/Account/FileAccount.Table.al

```diff
@@ -0,0 +1,55 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.FileSystem;
+
+/// <summary>
+/// A common representation of a file account.
+/// </summary>
+table 9450 "File Account"
+{
+    Extensible = false;
+    TableType = Temporary;
```

Comment:
Isnt `DrillDownPageId` and `LookupPageId` missing here?

Reply:
Normally I would agree. But the basic page is solved via the interface to be implemented.